src/store/admin/index.ts

```diff
@@ -126,11 +129,12 @@ const mod = {
       context: ActionContext<AdminState, RootState>,
       battleTag: string
     ): Promise<ProxySettings> {
-      const { commit, rootGetters, rootState } = moduleActionContext(context, mod);
+      const { commit, rootGetters  } = moduleActionContext(context, mod);
```

Comment:
Yeah the linting rules don't catch a lot of stuff. We recently replaced prettier with it, because there was no way to disable the line-length warning.